### ISSUE TITLE
"section section" is read before the section title

### ISSUE THREAD

**Issue by uskka** [Connexions/webview#2048]
When any text to speech app is used, and we open a new section, before section title is read, we can hear "section section". 
Here is a reproduction of this isse with Chromevox: 
https://drive.google.com/file/d/13iYS82RP8i98m4csTXUYV8jMrADXTQpw/view

**Comment by PiotrKozlowski** [comment]
It's caused because `header.coffee` is rerendered after it's loaded.
It looks like this:
1. header is loaded:
2. focus title
3. read `section - ...`
4. header is rerendered after first word was red
5. focus is lost
6. header is loaded again
7. focus is set to title
8. read `section - title`

I'm not sure why `header.coffee` is rendered multiple times. Also this happens only on first load on page. If you will select this page again later it will be loaded normally and `header` will be loaded only once.

**Comment by Stackblocks** [comment]
Unable to test because I don't have access to my accessibility machine today. Please test this. Thank you.
